source/collections/HashDictionary.ooc

```diff
@@ -20,17 +35,16 @@ HashDictionary: class {
 	}
 	merge: func (other: This) -> This {
 		result := this clone()
-	result _myHashBag myMap merge!(other _myHashBag myMap)
+		result _myHashBag myMap merge!(other _myHashBag myMap)
 		result
 	}
 	/* WARNING: The defaultValue parameter for Covers must be new Cell(cover) */
 	get: func <T> (key: String, defaultValue: T) -> T {
 		result := defaultValue
 		if (_myHashBag contains?(key)) {
-			storedType := _myHashBag getClass(key)
-			// is `T` a derived type or the same type as the stored type?
+			storedType := this _myHashBag getClass(key)
 			if (T inheritsFrom?(storedType)) {
-				result = _myHashBag getEntry(key, T) value as T
+				result = this _myHashBag getEntry(key, T) value as T
 			}
```

Comment:
Since you're cleaning this up, you might as well get rid of these unnecessary brackets...